siriuspy/siriuspy/fofb/main.py

```diff
@@ -703,6 +707,47 @@ def set_corr_accdec(self, option, value):
         self._update_log('...done!')
 
         return True
+    
+
+    def set_corr_accfilter(self, option, value):
+        """Set corrector accumulator filter."""
+        if self._corr_accfilter_enm != self._const.FilterOpt.Custom and \
+                option == 'value':
+            return False
+
+        if option == 'enum':
+            if value == self._const.FilterOpt.Unit:
+                filter = 20*[0.0]
+                filter[0] = 1
+                filter[5] = 1
+                filter[10] = 1
+                filter[15] = 1
+
+            elif value == self._const.FilterOpt.Switching:
+                b4 = [9.10339395e-01, -1.11484423e-16, 9.10339395e-01] # freq FOFB/4
+                a4 = [-1.11484423e-16, 8.20678791e-01] # freq FOFB/4
+                b2 = [0.83408931895964943947774372645654, 0.83408931895964943947774372645654, 0.0] # freq FOFB/2
+                a2 = [0.66817863791929887895548745291308, 0.0] # freq FOFB/2
+                filter = 20*[0.0] 
+                filter[0:3] = b4
+                filter[3:5] = a4
+                filter[5:8] = b2
+                filter[8:10] = a2
+            else:
+                filter = self._corr_accfilter_val
+            self._corr_accdfilter_enm = value
+            self.run_callbacks('FOFBAccFilter-Sts', value)
+            self.run_callbacks('FOFBAccFilter-SP', filter)
+        else:
+            filter = value
+        self._corr_accfilter_val = filter
+        self.run_callbacks('FOFBAccFilter-RB', filter)
+
+        self._update_log('Setting FOFB Acc filter...')
+        self._corrs_dev.set_fofbacc_filter(filter)
+        self._update_log('...done!')
```

Comment:
I think this code should then call `check_fofbacc_filter`, so we can have feedback on if it worked.

Reply:
By standard we don't check immediately, but there's a check thread ` _check_corrs_configs`.